docs/openapi/components/schemas/common/IntentToImport.yml

```
@@ -1,7 +1,7 @@
 $linkedData:
-  term: IntentToSell
-  '@id': https://w3id.org/traceability#IntentToSell
-title: Intent To Sell
+  term: IntentToImport
+  '@id': https://w3id.org/traceability#IntentToImport
+title: Intent To Import
 description: >-
   A simple case where an entity (person or organization) wishes to notify some
   third party that they intend to sell some product, likely into a different
```

This will nearly duplicate the second line, which has `in` where `it` should be. The originator of this PR should be able to correct that.
```suggestion
  third party that they intend to import some product, likely into a different
  geographic region than it originated in.  This is a highly simplified version
```